tools/cli-client/README.md

```diff
@@ -10,13 +10,13 @@ A CLI client that triggers GitHub workflows for building, uploading and releasin
 
 See ["How to Contribute as a Maintainer"](https://github.com/canonical/oci-factory?tab=readme-ov-file#as-a-maintainer--).
 
-Upon finishing the onboarding, the user should receive a GitHub Personal Access Token. This token grants you proper
-permissions to trigger the build, upload and release of your rocks. This token should never be shared with 
-third-parties, nor put anywhere that is publicly available.
+Upon finishing the onboarding, the user should add their GitHub username to the `rockcraft.yaml` file in the
+`maintainers` field of the corresponding rock. This field is a list of GitHub usernames that are allowed to
```

Comment:
Should we add a reference to the README.md with the sample contacts.yaml?

Reply:
Thx, added.